### ISSUE TITLE
[BUG] chia farm summary doesn't show correct information for slave harvest node

### ISSUE THREAD

**Issue by gades** [Chia-Network/chia-blockchain#6563]
**Describe the bug**
chia farm summary doesn't show correct information for slave harvest node

**To Reproduce**

1. Configure slave node by using this article https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines
2. execute command on slave node chia farm summary

Connection error. Check if wallet is running at 9256
Connection error. Check if full node is running at 8555
Connection error. Check if farmer is running at 8559
Farming status: Not available
Total chia farmed: Unknown
User transaction fees: Unknown
Block rewards: Unknown
Last height farmed: Unknown
Plot count: 16
Total size of plots: 1.584 TiB
Estimated network space: Unknown
Expected time to win: Unknown
Note: log into your key using 'chia wallet show' to see rewards for each key

**Expected behavior**
I'm expecting status - farming instead of Not available
Also, the better to show Expected time to win: instead of Unknown

**Desktop (please complete the following information):**

- OS: Linux
- OS Version/Flavor: Ubuntu 20.04.2 LTS
- CPU: Intel(R) Core(TM) i3-7100U CPU @ 2.40GHz

**Additional context**
The better to show total plots on master harvester because there it's showing only local and fix  expected time

**Comment by happycouak** [comment]
Duplicates and/or relates with https://github.com/Chia-Network/chia-blockchain/discussions/6467

**Comment by mariano54** [comment]
Fixed by 1.2.0

**Comment by happycouak** [comment]
Main node and harvesters have been upgraded to 1.2.0, all daemon/process restarted, still "chia farm summary" does not show up plots informations:

```
Plot count for all harvesters: 0
Total size of plots: 0.000 MiB
```


Is there something to do for make this to work, did I miss something ?
